src/components/input/checkbox/CheckboxComponent.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import { Checkbox } from "@mui/material";
+
+import { colors } from "../../colors/Colors";
+
+const CheckboxComponent = ({ disabled, variant }) => {
+  const [checked, setChecked] = useState(false);
+
+  useEffect(() => {
+    if (disabled) setChecked(false);
+  }, [disabled]);
+
+  switch (variant) {
+    case "control":
+      return (
+        <Checkbox
+          checked={checked}
+          sx={{
+            bgcolor: "transparent",
+            color: `${colors.white}`,
+            "&.Mui-checked": {
+              color: `${colors.emerald400}`,
+            },
+            "&:hover": {
+              bgcolor: "transparent",
+              color: `${colors.emerald400}`,
+            },
+          }}
+          disabled={disabled}
+          onClick={() => setChecked(!checked)}
+        />
+      );
+
+    default:
```

Comment:
do we need this default checkbox?

Reply:
I think we can remove the default checkbox after all the checkbox components are done? Noticed that we have a "simple checkbox" which hasn't been created yet, but if we go on that page, it will cause an error.